chartfx-chart/src/main/java/io/fair_acc/chartfx/utils/SimplePerformanceMeter.java

```diff
@@ -213,7 +214,7 @@ public void run() {
                 cpuLoadProcessAvgInternal = computeAverage(cpuLoadProcessInternal, cpuLoadProcessAvgInternal, alpha);
                 cpuLoadSystemAvgInternal = computeAverage(cpuLoadSystemInternal, cpuLoadSystemAvgInternal, alpha);
 
-                FXUtils.runFX(SimplePerformanceMeter.this::updateProperties);
+                Platform.runLater(SimplePerformanceMeter.this::updateProperties);
```

Comment:
Why not use an AnimationTimer?

Reply:
good point, also since there already is an animation timer, that can just also do the update.